tests/cases/compiler/erasableSyntaxOnly.ts

```diff
@@ -0,0 +1,55 @@
+// @erasableSyntaxOnly: true
+// @noEmit: true
+
+class MyClassErr {
+    // No parameter properties
+    constructor(public foo: string) { }
+}
+
+namespace IllegalBecauseInstantiated {
+    export const m = 1;
+}
+
+namespace AlsoIllegalBecauseInstantiated {
+    class PrivateClass {
+
+    }
+}

```

Comment:
```suggestion
}

namespace IllegalBecauseNestedInstantiated {
	namespace Nested {
		export const m = 1;
	}
}
```

Might be good to explicitly note that nesting needs to be checked too in case third parties read our tests and don't realize their "contains values" check has to be recursive.